tests/e2e/testdata/scenarios/powertool/logginglogmetric_clear_state_into_spec/mock_object_with_unmanaged_fields.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: logging.cnrm.cloud.google.com/v1beta1
```

Comment:
Maybe we could have a way to apply with a given manager, or force the managedFields, rather than mocking the get?

Reply:
Created a new PR that supports forcing spec fields with the manager `cnrm-controller-manager`: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2008